README.md

```diff
@@ -28,6 +28,24 @@ The development version of `metajam` package can also be installed from GitHub u
 devtools::install_github('NCEAS/metajam')
 ```
 
+## Workflow
+
+The process of using `metajam` is as follows:
+
+1. Get the URL for the dataset to download
+2. Download the data (`metajam::download_d1_data`)
+3. Read the data into R (`metajam::read_d1_files`)
```

Comment:
I would suggest `Read the data and metadata into R` one advantage of metajam is you can import both as an R object and for example query on units